src/gallia/command.py

```diff
@@ -226,9 +256,45 @@ def prepare_artifactsdir(
 
         raise ValueError("base_dir or force_path must be different from None")
 
-    @abstractmethod
-    def run(self, args: Namespace) -> int:
-        ...
+    def entry_point(self, args: Namespace) -> int:
+        if self.ARTIFACTSDIR:
+            self.artifacts_dir = self.prepare_artifactsdir(
+                args.artifacts_base,
+                args.artifacts_dir,
+            )
+            setup_logging(
+                self.get_log_level(args),
+                self.get_file_log_level(args),
+                self.artifacts_dir.joinpath("log.json.zst"),
+            )
+        else:
+            setup_logging(self.get_log_level(args))
+
+        exit_code = 0
+        try:
+            exit_code = self.run(args)
+        except (BrokenPipeError, ConnectionRefusedError) as e:
+            exit_code = 1
+            self.logger.critical(g_repr(e))
+        except KeyboardInterrupt:
+            exit_code = 128 + signal.SIGINT
+        # Ensure thet META.json gets written in the case a
+        # command calls sys.exit().
+        except SystemExit as e:
+            exit_code = e.code
+        except Exception:
+            exit_code = ExitCodes.UNHANDLED_EXCEPTION
+            traceback.print_exc()
+        else:
+            self.logger.info(f"Stored artifacts at {self.artifacts_dir}")
```

Comment:
Maybe we can move this to `finally`?

Reply:
I do not like it, since in every case this line is printed. That's pretty annyoing tbh.